src/site/content/en/payments/web-based-payment-apps-overview/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Web-based payment apps developers guide
+subhead: |
+  How to integrate your web-based payment app with Web Payments and provide a better user experience for customers.
+authors:
+  - agektmr
+date: 2020-07-17
+description: |
+  Learn how to adapt your web-based payment app to work with Web Payments and provide a better user experience for customers.
+tags:
+  - payments
+---
+
+[Web
+Payments](/empowering-payment-apps-with-web-payments/#what-is-web-payments)
+brings to the web a browser native interface that allows users to enter required
+payment information easier than ever before. The APIs can invoke web-based
+payment apps, as well as [native payment
+apps](/android-payment-apps-overview/).
+
+## Benefits of web-based payment apps
+
+<figure class="w-figure" style="width:300px; margin:auto;">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/payments/skip-the-sheet.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/payments/skip-the-sheet.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Checkout flow with a web-based payment app.
+  </figcaption>
+</figure>
+
+* Payments are made in modals, in the context of the merchant website, which
+  provides better user experience than typical payment app techniques that use
+  redirects or pop-ups.
+* Web Payments APIs can be integrated into established websites allowing you to
+  leverage the existing user base.
+* Unlike native apps, web-based payment apps don't need to be installed in
+  advance.
+
+## How does a web-based payment app work?
+
+Web-based payment apps are built using the standard web technologies. Every
+web-based payment app must include a service worker.
+
+[Service
+worker](https://developers.google.com/web/fundamentals/primers/service-workers)
+is an event-driven script that runs in the background even if the registering
+website is not opened in the browser. Service workers enable websites to work
+offline and send push notifications, because they can respond to requests with
+cache that is stored locally in advance.
+
+In a web-based payment app, a service worker can respond to payment requests:
+
+* Open a modal window and display the payment app's interface.
+* Bridge the communication between the payment app and the merchant.
+* Get an authorization from the customer and pass the payment credential to the
+  merchant.
+
+Learn how a payment app works on a merchant in [Life of a payment
+transaction](/life-of-a-payment-transaction/).
+
+## How merchants discover your payment app
+
+In order for a merchant to use your payment app, they need to use the [Payment
+Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) and
```

Comment:
There don't exist one yet. Perhaps this is a good timing because Firefox just started implementing this feature as well:
https://groups.google.com/g/mozilla.dev.platform/c/gBQp1URD1lE/m/Fswh-5-ZBgAJ